src/test/java/emissary/test/core/junit5/LogbackTesterTest.java

```diff
@@ -0,0 +1,99 @@
+package emissary.test.core.junit5;
+
+import ch.qos.logback.classic.Level;
+import org.junit.jupiter.api.Test;
+import org.opentest4j.AssertionFailedError;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Replace the wildcard import with individual method imports.

Reply:
Applied the changes.